src/nmodl/parse1.ypp

```diff
@@ -227,7 +227,7 @@ uniton: UNITSON {replacstr($1, "");}
 	;
 limits: /*nothing*/
 		{$$ = stralloc("", (char*)0);}
-	| LT real ',' real GT
+	| LT real_signed ',' real_signed GT
```

Comment:
would `| LT number ',' number GT` work?

Reply:
Yes, I just wanted to suggest the same. It does work.

Reply:
but there is no `0` added.

Reply:
done by the way